app/javascript/src/components/Dashboard/Contacts/Table/DropdownMenu.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+
+const DropdownMenu = () => <MenuHorizontal size={16} />;
```

Comment:
This abstraction into a separate component is not needed.

Reply:
Ok I have removed this. Thanks for pointing out.